src/main/java/com/sprint/mission/discodeit/service/file/FileUser.java

```diff
@@ -0,0 +1,133 @@
+package com.sprint.mission.discodeit.service.file;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.service.file.repository.FileUserRepository;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+
+public class FileUser implements FileUserRepository {
+
+  private static final String FILE_PATH = "src/main/java/com/sprint/mission/discodeit/service/file/data/user/user.json";
+  private static final ObjectMapper objectMapper = new ObjectMapper();
+
+  private List<User> readAllUsers() {
+    File file = new File(FILE_PATH);
+    if (!file.exists() || file.length() == 0)
+      return new ArrayList<>();
```

Comment:
if 문 안에는 {}을 붙여주는게 좀 더 가독성에 좋을 것 같아요.  
```suggestion
    if (!file.exists() || file.length() == 0) {
      return new ArrayList<>();
    }
```